relenv/build/common.py

```diff
@@ -1495,11 +1495,13 @@ def find_pythonlib(libdir):
             format_shebang("../../../bin/python3"),
         )
 
-    patch_shebang(
-        str(pymodules / "cgi.py"),
-        "#! /usr/local/bin/python",
-        format_shebang("../../bin/python3"),
-    )
+    # Moved in python 3.13 or removed?
+    if (pymodules / "cgi.py").exists():
+        patch_shebang(
+            str(pymodules / "cgi.py"),
+            "#! /usr/local/bin/python",
+            format_shebang("../../bin/python3"),
+        )
 
```

Comment:
I don't know if this matters, but the path to python on Windows is `Scripts\python3.exe`... maybe this only applies to Linux.

Reply:
Yes, I it only applies on linux. Windows has it's own finalize method in `relenv/build/windows.py`